nextcloudappstore/core/models.py

```diff
@@ -123,6 +125,13 @@ def _latest_non_nightly(self, releases):
             return None
 
 
+class AppAuthor(Model):
+    name = CharField(max_length=128, verbose_name=_('Full name'))
+    homepage = URLField(max_length=256, blank=True,
+                        verbose_name=_('Homepage'))
+    mail = EmailField(max_length=128, verbose_name=_('E-Mail'), blank=True)
```

Comment:
Right, however its already mail in the schema and throughout the entire code base xD

I think its fine as is :)